tests/FakeItEasy.Tests/Core/SequentialCallContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core
+{
+    using System.Collections.Generic;
+    using FakeItEasy.Core;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SequentialCallContextTests

```

Comment:
I wrote these tests as an afterthought, when I removed `OrderedFakeAsserter` and the associated tests (the test cases are just transposed from the ones in `OrderedFakeAsserterTests`). They're supposed to test `SequentialCallContext` in isolation (which is not what `OrderedCallMatchingSpecs` is doing). But the truth is that it's not really in isolation; it depends on `FakeManager` and a bunch of other things.

I could either change the internal API to make `SequentialCallContext` testable in isolation, or just drop these tests. Let me know which you prefer.